Create 2019-10-02-eskilstuna.adoc (#2510)
jenkins-infra/jenkins.io: content/_data/events/2019-10-02-eskilstuna.adoc

---

> name: "Jenkins Configuration as Code (a.k.a jcasc) and pipelines"

Reviewer: ```suggestion
title: "Eskilstuna Meetup: Jenkins Configuration as Code (a.k.a jcasc) and pipelines"
```

> title: "Eskilstuna Meetup: Jenkins Configuration as Code (a.k.a jcasc) and pipelines"

Reviewer: Most other events have listed the event name here and then placed the topic inside the body of the event.  Would you consider the same here so that the title is used more consistently.
```suggestion
title: "Eskilstuna DevOps Meetup"
```

The CONTRIBUTING doc says:

> Note, this is not subject of the event, but the name. Example: "Seattle JAM", "Jenkins World 2017". Basically, take a look at the events list on https://jenkins.io/events/ as though you were trying to choose events you would go to. "August JAM" is not specific enough, but "Jenkins World 2017" is.

location: "Eskilstuna, Sweden"
date: "2019-10-02T17:30:00"
link: "https://www.meetup.com/Eskilstuna-DevOps-Meetup/events/264725257/"
---

> It's time to talk about Jenkins Pipelines!

Reviewer: ```suggestion
Jenkins Configuration as Code (a.k.a jcasc) and pipelines

It's time to talk about Jenkins Pipelines!
```

Ewelina Wilkosz (Praqma, Malmö) will demonstrate Jenkins Configuration as Code plugin.
Alexandra Sedova (also Praqma, Stockholm) will share her experience with pipelines - she has been working with Jenkins pipelines quite some time and finds them a very powerful approach to creating delivery chains.

> Pipelines and jcasc togehter are an easy way to keep your complete CI as code.

Reviewer: ```suggestion
Pipelines and jcasc together are an easy way to keep your complete CI as code.
```
